prog/weave-kube/weave-monitoring-svc.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+   name: weave-net-monitoring
+   namespace: kube-system
+   annotations:
+     prometheus.io.scrape: "true"
```

Comment:
Just to note, this config is also for a case when `remote_write` is used, i.e. scrapping happens on a Prometheus instance running outside Service.